res/res_pjsip_endpoint_identifier_ip.c

```diff
@@ -241,6 +264,42 @@ static int header_identify_match_check(void *obj, void *arg, int flags)
 	return 0;
 }
 
+/*! \brief Comparator function for matching an object by request line */
+static int request_identify_match_check(void *obj, void *arg, int flags)
+{
+	struct ip_identify_match *identify = obj;
+	struct pjsip_rx_data *rdata = arg;
+
+	if (ast_strlen_zero(identify->match_request)) {
+		return 0;
+	}
+
+	int len;
+	char buf[PATH_MAX];
```

Comment:
Why PATH_MAX? PJSIP provides PJSIP_MAX_URL_SIZE for the maximum URI size

Reply:
Thank you, changed it.